releases/v0.2.0.md

```diff
@@ -0,0 +1,71 @@
+# Release Notes for v0.2.0
+Confidential Containers has adopted a six-week release cadence.
+This is our first release on this schedule.
+This release mainly features incremental improvements to our build system and tests
+as well as minor features, adjustments, and cleanup.
+
+Please see the [quickstart guide](../quickstart.md) for details on how to try out Confidential Containers
+
+## What's new
+
+- Kata CI uses existing Kata tooling to build components
+- Kata CI caches build environments for components
+- Incremental advances in signature support
+- KBS no longer required to use unencrypted images with SEV.
+
+## Hardware Support
+Confidential Containers is tested with attestation on the following platforms:
+- Intel TDX
+- AMD SEV
+
+The following platforms are untested or partially supported:
+- AMD SEV-ES
+- IBM Z SE
+
+The following platforms are in development:
+- Intel SGX
```

Comment:
We currently have a test for Intel SGX about to be merged on the Operator.
So, I think this should be "bumped" to, at least" a new bullet saying

```
Confidential Containers is tested without attestation  attestation on the following platforms:
- Intel SGX
```

/cc @dcmiddle @mythi 
```

Reply:
I assume - removing the duplicate ` attestation `?

Reply:
How about moving to it to the untested or partially supported category?

Reply:
I'm not the one running the Enclave CC show, but I think it can be consiudered supported and tested.

We have a test about to be merged on the Operator side, it's just lacking the attestation part.

Reply:
I understand that talking about exactly what is supported has the potential to be dramatic.

My goal with the sections is really just to have 1) totally supported and has pretty solid tests for most features; basically done except for new features that are added to the project as a whole 2) almost 1 3) we're working on it. My impression is that `enclave-cc` goes in the second category although there has been a lot of great progress lately. We can adjust the names of the sections if that would make things more clear. 

wdyt @dcmiddle @mythi

Reply:
Dramatic?!?! WHO IS GOING TO BE DRAMATIC?!?!?!?!?!
;-)
I'm actually fully onboard with listing SGX in level 2. We've got operator and some CI. We need to get more SGX-HW testing integrated. On the upside, though, @mythi got simulation-mode going which makes it easy to test without real SGX hardware. That way people can try it out on whatever platform they have access to.

Reply:
I was planning on being dramatic but maybe not.